main.tf

```
@@ -1 +1,193 @@
-# Your Terraform code goes here :)
+locals {
+  sqs_queue_name  = length(var.sqs_queue_name) > 0 ? var.sqs_queue_name : "${var.prefix}-sqs-${random_id.uniq.hex}"
+  s3_logs_location = "${var.s3_bucket_arn}/${var.aws_organization_id}/*AWSLogs/*"
```

Can we use the data source `"aws_organizations_organization"` to get the organization ID?
```
data "aws_organizations_organization" "main" {}
```
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/organizations_organization

Then we can fetch it like:
```suggestion
  s3_logs_location = "${var.s3_bucket_arn}/${data.aws_organizations_organization.main.id}/*AWSLogs/*"
```

---

Added both statements per your comments